src/main/kotlin/io/github/mojira/arisa/modules/Module.kt

```diff
@@ -47,8 +52,8 @@ fun <T> assertNotNull(e: T?) = when (e) {
     else -> Unit.right()
 }
 
-fun assertOr(vararg list: Either<OperationNotNeededModuleResponse, ModuleResponse>) =
-    if (list.any { it.isRight() }) {
+fun assertEither(vararg list: () -> Either<OperationNotNeededModuleResponse, ModuleResponse>) =
```

Comment:
Why was the parameter changed to a list of functions?
I don't see how any of the other changes would not work if this was kept as a list of Eithers

Reply:
I suppose the `any` method returns `true` once it finds an element matching the predicate, so by using lambda not all asset are executed.